connect/tests/src/test/java/com/telenor/connect/ui/ConnectLoginButtonTest.java

```diff
@@ -28,28 +36,41 @@
 
     @Before
     public void before() {
-        Whitebox.setInternalState(ConnectSdk.class, "sSdkInitialized", false);
+        Whitebox.setInternalState(ConnectSdk.class, "sdkProfile", (SdkProfile) null);
+        Whitebox.setInternalState(RestHelper.class, "wellKnownApiMap", WELL_KNOWN_API_MAP);
     }
 
     @Test(expected = ConnectNotInitializedException.class)
     public void clickingLoginButtonBeforeInitializingSdkThrows() {
         Activity activity = Robolectric.buildActivity(TestActivity.class).create().get();
+
         ConnectLoginButton button = (ConnectLoginButton) activity.findViewById(R.id.login_button);
         button.performClick();
     }
 
     @Test
-    public void clickingLoginButtonWithInitializedSdkStartsConnectActivity() {
+    public void clickingLoginButtonWithInitializedSdkStartsConnectActivityAndReturnsValidWellKnownConfig() {
         ConnectSdk.sdkInitialize(RuntimeEnvironment.application);
-        Activity activity = Robolectric.buildActivity(TestActivity.class).create().get();
-        ConnectLoginButton button = (ConnectLoginButton) activity.findViewById(R.id.login_button);
+        WELL_KNOWN_API_MAP.put(
+                ((ConnectSdkProfile) ConnectSdk.getSdkProfile()).getWellKnownEndpoint(),
+                MOCKED_VALID_WELL_KNOWN_API);
+
+        final Activity activity = Robolectric.buildActivity(TestActivity.class).create().get();
+        final ConnectLoginButton button = (ConnectLoginButton) activity.findViewById(R.id.login_button);
         button.setLoginScopeTokens("profile");
         button.performClick();
 
+        assertThat("Is activity started?",
```

Comment:
`assertThat("Is activity started?",` ->
`assertThat("Activity is started",`
would make more sense, I think. Reads like _assert that activity is started_.